modules/@apostrophecms/schema/ui/apos/logic/AposInputRelationship.js

```diff
@@ -185,10 +188,9 @@ export default {
         .filter(removeSelectedItem)
         .map(formatItems);
 
-      const suggestion = !qs.autocomplete && this.suggestion;
-      const hint = (!qs.autocomplete || !results.length) && this.hint;
-
-      this.searchList = [ suggestion, ...results, hint ].filter(Boolean);
+      this.searchSuggestion = !qs.autocomplete && this.suggestion;
+      this.searchHint = (!qs.autocomplete || !results.length) && this.hint;
```

Comment:
The Suggestion and Hint UI objects were originally part of the list of results. I separated these out so that I could cleanly pseudo focus search suggestions by index and not have to worry about accounting for the fake UI 'results'